contracts/TradeStand.sol

```diff
@@ -0,0 +1,86 @@
+pragma solidity 0.8.7;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol";
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol";
+import "./abstract/EmergencyWithdrawable.sol";
+
+contract TradeStand is ERC721Enumerable, ERC721URIStorage, Ownable, EmergencyRecover {
+    using Counters for Counters.Counter;
+
+    // The token URI must be the full URI in this contract.
+    string constant baseUri = "";
+
+    Counters.Counter public _tokenIdCounter;
+
+    constructor() ERC721("Polymon World Trade Stand", "TRADE") {}
+
+    //++++++++
+    // Public functions
+    //++++++++
+
+    function exists(uint256 tokenId) public view returns (bool) {
+        return _exists(tokenId);
+    }
+
+    function tokensOfOwner(address owner) public view returns (uint256[] memory) {
+        uint256 tokenCount = balanceOf(owner);
+        if (tokenCount == 0) {
+            return new uint256[](0);
+        } else {
+            uint256[] memory result = new uint256[](tokenCount);
+            for (uint256 index; index < tokenCount; index++) {
+                result[index] = tokenOfOwnerByIndex(owner, index);
+            }
+            return result;
+        }
+    }
+
+    //++++++++
+    // Owner functions
+    //++++++++
+
+    function mint(string calldata metaFileURI) public onlyOwner returns (uint256) {
+        _tokenIdCounter.increment();
```

Comment:
Why use counter here and not a simple ++

Reply:
This is how it was in the contract I received at the beginning. But I see you know that a simple ++ would be more my style.